khamraevamir/money-transfer
# Config: setup Django project structure and basic settings

# Description
Initial Django project setup and configuration for Transaction Management System

## Changes made:
- Created Django project with 'config' as main module
- Added 'core' application
- Configured base settings:
  - Static/Media files
  - Time zone (Asia/Tashkent)
  - Language (ru-ru)
  - Basic URL patterns
  - Database (SQLite for now)
- Added basic URL configuration
- Setup project structure:
  - backend/
    - config/
    - core/
    - manage.py
    - requirements.txt

## How to test
1. Create and activate virtual environment: 
```
python3 -m venv venv
source venv/bin/activate
```
2. Install dependencies:
```
pip install -r requirements.txt
```
3. Run migrations:
```
python manage.py migrate
```
4. Run server:
```
python manage.py runserver
```
5. Check if server runs at http://127.0.0.1:8000/admin
